src/index.css

```diff
@@ -1,27 +1,121 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;1,500&display=swap");
+
 body {
+  background-color: #fa5091;
+  font-family: "Poppins", sans-serif;
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  color: white;
+}
+
+/* width */
+::-webkit-scrollbar {
+  width: 10px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  background: #f1f1f1;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: #888;
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: #555;
+}
+
+.sub-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  margin-top: 3em;
+}
+
+.country-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 20em;
+  height: 20em;
+  margin: 0.2em;
+  border: none;
+  border-radius: 0.3em;
+  cursor: pointer;
+}
+
+.country-container img {
+  width: 15em;
+  height: 10em;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0 0 10em;
+}
```

Comment:
- [ ] The original design corresponds to a mobile version so please make sure that the mobile version looks well for a better user interface, we could use two columns like the original design and order the content on the details page.

| Original design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/218936201-b15e0717-a2e1-40f0-96ce-76b1422de559.png) | ![image](https://user-images.githubusercontent.com/58642949/218936274-ce4d05fc-01ad-4758-b4ba-b8eb1088486d.png) |
| ![image](https://user-images.githubusercontent.com/58642949/218936329-83907390-6d5e-44da-8bc0-8f0f980a7187.png) | ![image](https://user-images.githubusercontent.com/58642949/218936395-1307c02d-add6-494a-8f96-9c1b27c1256d.png) |